packages/trader/src/Modules/Reports/Containers/statement.jsx

```diff
@@ -197,32 +201,36 @@ class Statement extends React.Component {
                 ) : (
                     <>
                         <DesktopWrapper>
-                            <DataTable
-                                className='statement'
-                                data_source={data}
-                                columns={this.columns}
-                                onScroll={handleScroll}
-                                getRowAction={row => this.getRowAction(row)}
-                                is_empty={is_empty}
-                                custom_width={'100%'}
-                                getRowSize={() => 63}
-                                content_loader={ReportsTableRowLoader}
-                            >
-                                <PlaceholderComponent is_loading={is_loading} />
-                            </DataTable>
+                            {this.show_loading_placeholder ? (
```

Comment:
this condition is checked in both `MobileWrapper` and `DesktopWrapper`. Can we change this to something like below?
    
    {this.show_loading_placeholder ? (
        ...
    ) : (
        <React.Fragment>
            <DesktopWrapper></DesktopWrapper>
            <MobileWrapper></MobileWrapper>
        </React.Fragment>
    )}

Reply:
Yea why not.